homeassistant/components/kraken/strings.json

```
@@ -0,0 +1,25 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "[%key:common::config_flow::abort::single_instance_allowed%]"
+        },
+        "error": {},
+        "step": {
+            "user": {
+                "data": {},
+                "description": "[%key:common::config_flow::description::confirm_setup%]"
+            }
+        }
+    },
+    "options": {
+        "step": {
+            "init": {
+                "data": {
+                    "scan_interval": "Update interval",
+                    "tracked_asset_pairs": "Tracked Asset Pairs"
+                },
+                "description": "Configure options for Kraken"
```

We can leave this out. 
```suggestion
                }
```